README.md

```diff
@@ -25,11 +26,13 @@ Do not use these apps unless you specifically know what you are doing.
 2. Add the ios targets `rustup target add aarch64-apple-ios x86_64-apple-ios`
 3. `cargo install cargo-lipo`
 4. `cargo install cbindgen`
-5. `sudo gem install cocoapods` to install cocoapods
+5. Install Ruby dev and gems (`sudo apt install -y rubygems ruby-dev`)
+6. `sudo gem install cocoapods` to install cocoapods
 
 ### Building
-1. In the `zecwallet-mobile/rust/ios` directory, run `./build.sh`. This step will take a long time.
+1. In the `rust/ios` directory, run `./build.sh`.
+   This step will take a long time.
 2. In the `zecwallet-mobile/ios` directory, run `pod install`
```

Comment:
While we're trimming zecwallet, this looks like a good target too.

Reply:
it will be